org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/DashboardController.java

```diff
@@ -763,6 +768,7 @@ private void asyncRenderComponents(MDashboardContent dashboardContent, Dashboard
 			}
     		int thisClientId = Env.getAD_Client_ID(Env.getCtx());
     		if((thisClientId == 0 && systemAccess) || thisClientId != 0) {
+    			addDrillAcrossEventListener(AD_Process_ID, appDesktop);
```

Comment:
this should only be needed if (dashboardContent.isEmbedReportContent()), shouldn't goes here.